anyway/infographics_utils.py

```diff
@@ -61,768 +144,678 @@ def serialize(self):
         return output
 
 
-def extract_news_flash_location(news_flash_id):
-    news_flash_obj = db.session.query(NewsFlash).filter(NewsFlash.id == news_flash_id).first()
-    if not news_flash_obj:
-        logging.warning(f"could not find news flash id {str(news_flash_id)}")
-        return None
-    resolution = news_flash_obj.resolution if news_flash_obj.resolution else None
-    if not news_flash_obj or not resolution or resolution not in resolution_dict:
-        logging.warning(f"could not find valid resolution for news flash id {str(news_flash_id)}")
-        return None
-    data = {"resolution": resolution}
-    for field in resolution_dict[resolution]:
-        curr_field = getattr(news_flash_obj, field)
-        if curr_field is not None:
-            data[field] = curr_field
-    gps = {}
-    for field in ["lon", "lat"]:
-        gps[field] = getattr(news_flash_obj, field)
-    return {"name": "location", "data": data, "gps": gps}
-
-
-def get_query(table_obj, filters, start_time, end_time):
-    query = db.session.query(table_obj)
-    if start_time:
-        query = query.filter(getattr(table_obj, "accident_timestamp") >= start_time)
-    if end_time:
-        query = query.filter(getattr(table_obj, "accident_timestamp") <= end_time)
-    if filters:
-        for field_name, value in filters.items():
-            if isinstance(value, list):
-                values = value
-            else:
-                values = [value]
-            query = query.filter((getattr(table_obj, field_name)).in_(values))
-    return query
-
+class AccidentCountBySeverityWidget(Widget):
+    widget_id: WidgetId = dataclasses.field(init=False, default=WidgetId.accident_count_by_severity)
 
-def get_accident_count_by_accident_type(location_info, start_time, end_time):
-    all_accident_type_count = get_accidents_stats(
-        table_obj=AccidentMarkerView,
-        filters=location_info,
-        group_by="accident_type_hebrew",
-        count="accident_type_hebrew",
-        start_time=start_time,
-        end_time=end_time,
-    )
-    merged_accident_type_count = [{"accident_type": "התנגשות", "count": 0}]
-    for item in all_accident_type_count:
-        if "התנגשות" in item["accident_type"]:
-            merged_accident_type_count[0]["count"] += item["count"]
-        else:
-            merged_accident_type_count.append(item)
-    return merged_accident_type_count
+    def __init__(self, request_params: RequestParams):
+        Widget.__init__(self, request_params)
```

Comment:
Why not the standard `super().__init__(request_params)`?

Reply:
All the calls to super in our project are of the form 
`super(<super class name>, self)__init__(...`, while 
`Widget.__init__(self, ...` 
is shorter.

Reply:
This is the old way. `super().__init__(...)` is even shorter, does the right thing, and does not hard code the name of the base class.